receiver/zookeeperreceiver/scraper.go

```diff
@@ -77,45 +77,57 @@ func (z *zookeeperMetricsScraper) scrape(ctx context.Context) (pmetric.Metrics,
 	var ctxWithTimeout context.Context
 	ctxWithTimeout, z.cancel = context.WithTimeout(ctx, z.config.Timeout)
 
+	response, err := z.runCommand(ctxWithTimeout, "mntr")
+	if err != nil {
+		return pmetric.NewMetrics(), err
```

Comment:
Why not just `return response, err`

Reply:
The pmetric struct is not a pointer or interface so it must be a concrete value. Unfortunately, I think it would require a change in [`scraperhelper`](https://github.com/open-telemetry/opentelemetry-collector/blob/ce6535092daee23d1d7492844dee639a24cd803d/receiver/scraperhelper/scraper.go#L29) in order to allow it.